code/game/objects/items/cigs_lighters.dm

```diff
@@ -205,7 +206,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		e.start()
 		qdel(src)
 		return
-	// allowing reagents to react after being lit
```

Comment:
minor nitpick: dont delete even comments if you dont have to.
But this is in a merge-able state now.